lib/plugins/create/create.js

```diff
@@ -177,7 +177,7 @@ class Create {
     } else if ('template-path' in this.options) {
       // Copying template from a local directory
       const servicePath = this.options.path
-        ? untildify(this.options.path)
+        ? path.resolve(this.options.path)
```

Comment:
```suggestion
        ? path.resolve(process.cwd(), untildify(this.options.path))
```

Reply:
thank u for you suggestions. i tested ~/, it works well when i use path.resolve